lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -324,8 +327,8 @@ def decrement_positions_on_higher_items(position)
 
         # This has the effect of moving all the lower items up one.
         def decrement_positions_on_lower_items(position=nil)
-          return unless in_list?
-          position ||= send(position_column).to_i
+          position ||= Integer(send(position_column))
```

Comment:
`Integer(nil)` raises an error vs `nil.to_i` that returns `0`. Could this be a problem in some cases?

Reply:
It definitely is. Fixed.